docs/excel/custom-functions-data-types-concepts.md

```diff
@@ -1,19 +1,21 @@
 ---
-title: Custom functions and data types core concepts
-description: 'Learn the core concepts for using Excel data types with your custom functions.'
-ms.date: 11/03/2021
+title: Custom functions and data types
+description: 'Use Excel data types with your custom functions and Office Add-ins.'
+ms.date: 12/27/2021
 ms.topic: conceptual
 ms.custom: scenarios:getting-started
-ms.localizationpriority: medium
+ms.localizationpriority: high
 ---
 
-# Custom functions and data types core concepts (preview)
+# Use data types with custom functions in Excel (preview)
 
 [!include[Custom functions and data types availability note](../includes/excel-custom-functions-data-types-note.md)]
 
-Data types enhance the Excel JavaScript API by expanding support for data types beyond the original four (string, number, boolean, and error). Data types include support for formatted number values, web images, entity values, and arrays within entity values. Custom functions accept data types as both input and output values, expanding the calculation power of custom functions.
+Data types expand the Excel JavaScript API to support data types beyond the original four (string, number, boolean, and error). Data types include support for web images, formatted number values, entity values, and arrays within entity values.
 
-To learn more about using data types with an Excel add-in, see [Excel data types core concepts](excel-data-types-concepts.md).
+These data types amplify the power of custom functions, because custom functions accept data types as both input and output values. You can generate data types through custom functions, or take existing data types as function arguments into calculations. Once the JSON schema of a data type is set, this schema is maintained throughout custom function calculations.
```

Comment:
I think you can remove this redundancy without sacrificing clarity.
```suggestion
These data types amplify the power of custom functions, because custom functions accept data types as both input and output values. You can generate data types through custom functions, or take existing data types as function arguments into calculations. Once the JSON schema of a data type is set, this schema is maintained throughout the calculations.
```